src/constants.ts

```diff
@@ -288,6 +288,7 @@ WantedBy=graphical.target
 `);
 
 export const SATELLITE_SERVICE = 'register-satellite';
+export const AAP_SERVICE = 'ansible-controller-registration';
```

Comment:
We take care of this in `image-builder-crc`, so this breaks the image build.
If you really want to add this to the list of services, it should be (see here [line](https://github.com/kingsleyzissou/image-builder/blob/5204665cf9de6bf22ce4591fa2cc976b366bc4b4/internal/v1/handler_compose_image.go#L29) and [here](https://github.com/kingsleyzissou/image-builder/blob/5204665cf9de6bf22ce4591fa2cc976b366bc4b4/internal/v1/handler_compose_image.go#L1306)):
`aap-first-boot-reg.service`

Reply:
Yep, I can confirm I just tested this PR and it is erroring out with: `Failed to enable unit, unit ansible-controller-registration.service does not exist`

We should move all the code logic into the backend, UI should not be aware of any details here, it must use the API from the CRC.